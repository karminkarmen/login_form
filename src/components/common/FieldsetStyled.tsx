import styled from 'styled-components';

const FieldsetStyled = styled.fieldset`
    margin: unset;
    padding: unset;
    border: none;
    width: ${props => props.theme.width.mainElementWidth};
    height: ${props => props.theme.width.mainElementWidth};
`;

export default FieldsetStyled;
