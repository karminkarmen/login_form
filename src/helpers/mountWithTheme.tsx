import React from 'react';
import { mount } from 'enzyme';
import { ThemeProvider } from 'styled-components';
import theme from '../configs/theme';

const mountWithTheme = (child: React.ReactElement) => {
    return mount(child, {
        wrappingComponent: ({ children }) => <ThemeProvider theme={theme}>{children}</ThemeProvider>,
    });
}

export default mountWithTheme;