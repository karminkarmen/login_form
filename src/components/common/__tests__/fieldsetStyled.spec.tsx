import React from 'react';
import FieldsetStyled from '../FieldsetStyled';
import mountWithTheme from '../../../helpers/mountWithTheme';

describe('<FieldsetStyled />', () => {

    it('matches snapshot', () => {
        const wrapper = mountWithTheme(<FieldsetStyled />);
        expect(wrapper.html()).toMatchSnapshot();
    });

});
