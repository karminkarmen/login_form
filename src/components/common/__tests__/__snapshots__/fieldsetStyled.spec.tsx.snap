// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FieldsetStyled /> matches snapshot 1`] = `"<fieldset class=\\"sc-AxjAm hUOmCs\\"></fieldset>"`;
